src/components/Calendar/Events.js

```diff
@@ -0,0 +1,63 @@
+export default [
```

Comment:
I don't think this data needs to live in a separate file.

Reply:
I removed it and added the data to the Calendar.js file temporarily